release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -116,6 +116,11 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 
 You can now remove or rotate the GCP admin-level credential that the xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator] uses in Mint mode. This option requires the presence of the admin-level credential during installation, but the credential is not stored in the cluster permanently and does not need to be long-lived.
 
+[id="ocp-4-7-compliance-cis-benchmark"]
+==== CIS Kubernetes Benchmark profile for Compliance Operator
+
+You can now use the Compliance Operator to perform Center for Internet Security (CIS) Kubernetes benchmark checks. The CIS profile performs both Platform and Node scans against CIS Kubernetes benchmark standards and can apply remediations. 
```

Comment:
This might be a bit of terminology, but the CIS profile is just /based/ on the upstream kubernetes CIS standard, but it's different. In addition, there are two profiles, one for the Platform and one for Nodes. Could we say something like 'CIS profiles for OpenShift based on the CIS Kubernetes checks' ?